app-emacs/flycheck/flycheck-32_p20220328.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS="24.3"
+
+inherit elisp
+
+DESCRIPTION="Modern on-the-fly syntax checking extension for GNU Emacs"
+HOMEPAGE="https://www.flycheck.org/"
+COMMIT="3b5b4248074f016922c2674789d4a242528cf4c7"
+SRC_URI="https://github.com/flycheck/flycheck/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="test" # test requires cask and ert-runner which are not packaged yet
+
+RDEPEND=">=app-emacs/dash-2.12.1
+	>=app-emacs/pkg-info-0.4"
```

Comment:
There's nothing wrong with this, but since we don't have such old versions of dash and pkg-info you don't have to specify the version.